scripts/remove-emojis.js

```
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+
+/**
+ * Script to remove all emojis from the codebase
+ * Per CLAUDE.md: NO EMOJIS in professional communication
+ */
+
+import fs from 'fs';
+import path from 'path';
+
+// Comprehensive emoji regex pattern
+const emojiRegex = /[\u{1F600}-\u{1F64F}\u{1F300}-\u{1F5FF}\u{1F680}-\u{1F6FF}\u{1F1E0}-\u{1F1FF}\u{2600}-\u{26FF}\u{2700}-\u{27BF}\u{1F900}-\u{1F9FF}\u{1F018}-\u{1F270}\u{238C}-\u{2454}\u{20D0}-\u{20FF}\u{FE00}-\u{FE0F}\u{E0020}-\u{E007F}\u{1F191}-\u{1F251}]/gu;
```

Biome (2.1.2)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)